gator.Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.17-alpine AS builder
+
+COPY . /tmp/gatekeeper
+
+RUN apk add --no-cache \
+    upx
+
+WORKDIR /tmp/gatekeeper/cmd/gator
+
+RUN go get -d -v
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags="-w -s" -o /gator
+RUN upx --best --lzma /gator
```

Comment:
Since `upx` appears to embed extraction code which is subject to the GNU GPL, we probably want to avoid using it so that we are clearly in the "allow list" established by CNCF:

https://github.com/cncf/foundation/blob/main/allowed-third-party-license-policy.md

Reply:
Aah, that makes sense - I've removed it